conf/obj.h

```diff
@@ -677,6 +677,8 @@ struct m0_conf_rack {
 /** Hardware resource --- enclosure. */
 struct m0_conf_enclosure {
 	struct m0_conf_obj    ce_obj;
+	/** The node this controller is associated with. */
```

Comment:
copy & paste error:
s/this controller/this enclosure/

Reply:
Done